src/components/pages/_gridExample/gridExample.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Why are we adding _ in the front of the file name. I'm not sure if we should do this to just differentiate whether this is a sample code.

Reply:
Yes, it was an effort to differentiate. So its easy to find sample code and delete it if you want to fork from our branch and don't want the extra stuff there. 

As we add more walkthroughs and more sample code to go with them, I wanted to be able to give the files good names without having to always use example or sample at the beginning... and if the thing that makes it look like an example isn't at the beginning, then its harder to find to remove if desired.

I felt like it was valuable to have the samples in the code so they could be debugged and understood that way... without having to recreate using the steps in the walkthroughs. It also helps verify that the walkthrough steps actually work and are complete.

I'm open to other ideas.